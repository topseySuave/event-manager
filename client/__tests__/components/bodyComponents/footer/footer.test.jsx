import React from 'react';
import { shallow } from 'enzyme';
import Footer from
  '../../../../src/components/bodyComponents/footer/footer';

describe('Footer component', () => {
  const wrapper = shallow(<Footer />);
  test('Footer component should mount', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
