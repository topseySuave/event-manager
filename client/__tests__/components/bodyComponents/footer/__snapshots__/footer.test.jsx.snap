// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer component should mount 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="center-align"
        >
          <p>
            © 
            2018
            , All rights reserved.
            <Link
              href="/"
              replace={false}
              to="/"
            >
               
              Boots Events Manager
            </Link>
          </p>
        </div>
      </div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="center-align"
        >
          <p>
            © 
            2018
            , All rights reserved.
            <Link
              href="/"
              replace={false}
              to="/"
            >
               
              Boots Events Manager
            </Link>
          </p>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            © 
            2018
            , All rights reserved.
            <Link
              href="/"
              replace={false}
              to="/"
            >
               
              Boots Events Manager
            </Link>
          </p>,
          "className": "center-align",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "© ",
              2018,
              ", All rights reserved.",
              <Link
                href="/"
                replace={false}
                to="/"
              >
                 
                Boots Events Manager
              </Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "© ",
            2018,
            ", All rights reserved.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  " ",
                  "Boots Events Manager",
                ],
                "href": "/",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Array [
                " ",
                "Boots Events Manager",
              ],
              "type": [Function],
            },
          ],
          "type": "p",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="center-align"
          >
            <p>
              © 
              2018
              , All rights reserved.
              <Link
                href="/"
                replace={false}
                to="/"
              >
                 
                Boots Events Manager
              </Link>
            </p>
          </div>
        </div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="center-align"
          >
            <p>
              © 
              2018
              , All rights reserved.
              <Link
                href="/"
                replace={false}
                to="/"
              >
                 
                Boots Events Manager
              </Link>
            </p>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              © 
              2018
              , All rights reserved.
              <Link
                href="/"
                replace={false}
                to="/"
              >
                 
                Boots Events Manager
              </Link>
            </p>,
            "className": "center-align",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "© ",
                2018,
                ", All rights reserved.",
                <Link
                  href="/"
                  replace={false}
                  to="/"
                >
                   
                  Boots Events Manager
                </Link>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "© ",
              2018,
              ", All rights reserved.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    " ",
                    "Boots Events Manager",
                  ],
                  "href": "/",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Boots Events Manager",
                ],
                "type": [Function],
              },
            ],
            "type": "p",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
